photoeditor/src/main/java/ja/burhanrashid52/photoeditor/PhotoEditor.java

```diff
@@ -358,6 +358,16 @@ public Builder setPinchTextScalable(boolean isTextPinchScalable) {
         public PhotoEditor build() {
             return new PhotoEditorImpl(this);
         }
+
+        /**
+         * Set true true to clip the drawing brush to the source image.
+         *
+         * @param clip a boolean to indicate if brush drawing is clipped or not.
+         */
+        public Builder setClipSourceImage(boolean clip) {
+            parentView.setClipSourceImage(clip);
```

Comment:
Why setting it from `Builder` ? This should be done when inside `PhotoEditorImpl` constructor when `build()` method is called. What I am missing here ?

Reply:
You're right, it's cleaner this way. It's done now.